src/member.js

```diff
@@ -0,0 +1,75 @@
+const BN = require("bn.js")
+const {utils: { isAddress }} = require("web3")
+
+module.exports = class MonoplasmaMember {
+    constructor(name, address, earnings, active) {
+        this.name = name || ""
+        this.address = MonoplasmaMember.validateAddress(address)
+        this.earnings = earnings ? new BN(earnings) : new BN(0)
+        this.active = active || true
+    }
+
+    getEarningsAsString() {
+        return this.earnings.toString(10)
+    }
+
+    getEarningsAsInt() {
+        return this.earnings.toNumber()
+    }
+
+    addRevenue(amount) {
+        this.earnings = this.earnings.add(new BN(amount))
+    }
+
+    isActive() {
+        return this.active
```

Comment:
I'd `!!this.active` here too. Ensuring you're always working with booleans is cheap (just add `!!`) and can sidestep a bunch of [common JS gotchas](https://www.sitepoint.com/javascript-truthy-falsy/) regarding truthiness & falsiness.

Reply:
Note: feel free to ignore these comments about `!!` since it's not critical to this PR. They're only tangentially related nitpicks.